test/unit/lib/plugins/aws/deployFunction.test.js

```diff
@@ -421,6 +430,218 @@ describe('AwsDeployFunction', () => {
         })
       ).to.be.equal(true);
     });
+
+    describe('#KMSKeyArn should be set by "kmsKeyArn" or "awsKmsKeyArn"', () => {
```

Comment:
It would be great if all newly introduced tests will be configured with `runServerless` util as described: https://github.com/serverless/serverless/blob/master/test/README.md#unit-tests

Reply:
I've been going through the documentation for a little while now and I'm a little embarrassed but I just don't get it. Do you think you could provide me an example for one of these tests and I can take that and run with it?

Reply:
No worries :raised_hands: You can find some examples here: https://github.com/serverless/serverless/blob/master/test/unit/lib/plugins/aws/package/compile/functions.test.js#L1444

The general idea behind `runServerless` is to test outputs/results of certain functionalities instead of testing internal implementation, as it often makes it harder to refactor at later point in time. In case of `deployFunction` use of `runServerless` might be a bit tricky, as you'll most likely have to mock AWS calls. You can see an example of how single function deploy can be tested with `runServerless` - https://github.com/serverless/serverless/pull/8315/files

Please let me know if I can help more here :100:

Reply:
Let's move them to a top-level `describe` block in this test file as is suggested in https://github.com/serverless/serverless/blob/master/test/README.md#unit-tests

You might also want to rebase on top of master, because I believe we gained a few tests that rely on `runServerless` in this module in the meantime